certbot/docs/using.rst

```diff
@@ -57,10 +57,11 @@ standalone_ Y    N    | Uses a "standalone" webserver to obtain a certificate.
                       | domain. Doing domain validation in this way is
                       | the only way to obtain wildcard certificates from Let's
                       | Encrypt.
-manual_     Y    N    | Helps you obtain a certificate by giving you instructions to  http-01_ (80) or
-                      | perform domain validation yourself. Additionally allows you   dns-01_ (53)
-                      | to specify scripts to automate the validation task in a
-                      | customized way.
+manual_     Y    N    | Obtain a certificate by manually following instructions to    http-01_ (80) or
+                      | perform domain validation yourself. Certificates created this dns-01_ (53)
+                      | way do not support autorenewal.
+                      | Autorenewal may be enabled by specifying scripts to automate
```

Comment:
I generally like this, but how do you feel about either using the same name as we use to refer to it elsewhere, or just literally saying the flag name? To make it easier to cross-reference.

Reply:
Eeeeeeeee you're right. This isn't the only place I'm being inconsistent.

So, what's the real name? 

In the documentation, the header is *Pre and Post Validation Hooks*.

Personally I always call them authentication hooks, but I am willing to fix that habit, if we're calling them validation hooks.

Reply:
I think we call them authentication hooks more often, given a quick control-f around the docs, and looking through the implementation. But either way I do like using the name that matches the flag name, if not the flag name itself.